findPackage.js

```diff
@@ -23,7 +23,10 @@ const findPackage = function(packageName, searchPath) {
         } catch (err) {
           return;
         }
-        if (pkg.name != packageName) return;
+        if (pkg.name != packageName) {
+            console.log(`wrong package name in ${path.basename(file)}: actual=${pkg.name}, expected=${packageName}`);
```

Comment:
`${path.basename(file)}` is 'package.json` here.

I suppose you want to log the full path of the `file` (`${dirname}/${basename}`)?

Reply:
Yeah, sure. It would be better if found more than one `package.json`. But I prefer to use `path.join(dirname, basename)` for support correct OS-related paths